app/examples/vimium

```diff
@@ -9,26 +9,93 @@ set windowtitle=title
 set search=https://google.com/search?q=%s
 set searchwords=w~https://www.wikipedia.org/w/index.php?title=Special:Search&search=%s
 
+" Custom Commands
+command! enterInsertMode <toInsertMode>
+command! enterVisualMode <p.start><p.startVisualSelect>
+command! focusInput <insertAtFirstInput>
+command! LinkHints.activateMode <startFollowCurrentTab>
+command! LinkHints.activateModeToOpenInNewTab <startFollowNewTab><toLastUsedTab>
```

Comment:
What exactly is this supposed to do? It doesn't seem to work here. You might be looking for the `follownewtabswitch` setting. If so, please use that in the example instead of this mapping.

Reply:
Yes this is supposed to switch to the followed tab. I have changed it in the following commit.